package.xml

```diff
@@ -41,7 +42,6 @@
 
   <!-- Visualization dependencies -->
   <exec_depend>meshcat</exec_depend>
```

Comment:
Meshcat is not available as a ROS package

Reply:
Fair enough...